README-developers.md

```diff
@@ -10,3 +10,113 @@
 
 ### productivity
 - set up git [aliases](http://githowto.com/aliases)
+-
+### Branch and Release Management
+
+The latest development code lives in the `master` branch. It should be the
+primary branch for the repository. The primary branch will be the default branch
+checked out when user does `git clone`. Pull requests are merged directly to the
+`master` branch.
+
+#### Public Testing Servers
+
+We are planing to setup GitHub actions to deploy all merges to the testing
+server. Users are invited to test the latest changes on the testing server.
+Volunteers become our quality assurance team. They help us detect bugs before
+changes reach the production servers.
+
+#### Deployment to the Production Servers
+
+The component lead tags the testing image with a production tag when they think
+the testing deployment has had enough testing. The lead then deploys the new
+production image to the production servers.
+
+### Development Work in a Personal Fork Repository
+
+#### Cloning the Primary Repository
+
+You can clone the primary repository normally. This document shows example
+commands for this repository.
+
+`git clone https://github.com/online-bridge-hackathon/common-docs.git`
+
+#### Creating a Personal Fork
+
+Go to the component page in the GitHub website. You can follow the
+[Forking Projects](https://guides.github.com/activities/forking/) guide.
+
```

Comment:
Any advice needed on naming forks?

Reply:
I don't think there is any need for naming rules. They are personal repositories which would mean developers can freely manage naming as they wish.